src/bech32.cpp

```diff
@@ -370,11 +370,13 @@ std::string Encode(Encoding encoding, const std::string& hrp, const data& values
 }
 
 /** Decode a Bech32 or Bech32m string. */
-DecodeResult Decode(const std::string& str) {
+DecodeResult Decode(const std::string& str, bool silent) {
+    std::size_t max_size = silent ? 1024 : 90;
```

Comment:
Note to self, this should be 1023

Reply:
Wrap this function in a more generic "Decode" function that doesn't have size restrictions, possibly use an ENUM for indicating what sort of error guarantees you want.